weekly_report/generate_report.py

```diff
@@ -3,14 +3,16 @@
 from email.mime.text import MIMEText
 from email.mime.application import MIMEApplication
 from email.mime.multipart import MIMEMultipart
-
+from datetime import datetime
```

Comment:
Put `datetime` in its own block, and all `email.mime` libraries with the other 3rd party libraries

Reply:
Did some research it is a built in library.